### DISCUSSION TITLE
How to command LEDS over WIFI

### DISCUSSION THREAD

**Discussion by SturdyFool10** [PlummersSoftwareLLC/NightDriverStrip#182] [Q&A]
Hello there, I have started on making a (eventually) ESP based led strip that syncs over wifi, however, I have run into a bit of a wall and my searches are getting me nowhere, mainly, I am looking for a c++ framework suitable for commanding the strips through the internet but I could not find documentation on a recommended framework for sending such packets, nor could I find one

**Comment by robertlipe** [comment]
I think your wish was granted just a few days after youo posted this. See
https://github.com/PlummersSoftwareLLC/NightDriverStrip/discussions/186

**Comment by zephyrr** [comment]
I don't want to detract from the subject here, but Night Driver is a pretty huge and capable software base.  You might be able to get by for simpler projects with the much simpler (and lower level) FastLED library.

I am not in any way trying to discourage your participation here, just wanted you to be aware of alternative tools for different projects.
